Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py

```diff
@@ -124,6 +124,7 @@ def fetch_incidents_open_cases(client: Client, max_results: int, last_run: Dict[
                                first_fetch_time: Optional[int]
                                ) -> Tuple[Dict[str, int], List[dict]]:
     last_fetch = last_run.get('last_fetch', None)
+    case_anomaly = demisto.params().get('fetch_incident_cases')
```

Comment:
```suggestion
    case_anomaly = demisto.params().get('fetch_incident_cases') or 'Case Per Anomaly'
```

Reply:
Changed

Reply:
please follow the suggestion, it is a safer way to set the default value

Reply:
Changed